queries.sql

```diff
@@ -1,3 +1,18 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+/* Find all animals whose name ends in "mon". */
+SELECT * FROM animals WHERE name LIKE '%mon';
+/* List the name of all animals born between 2016 and 2019.*/
+SELECT DISTINCT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' and '2020-12-30'  
+/*List the name of all animals that are neutered and have less than 3 escape attempts. */
+SELECT DISTINCT name FROM animals WHERE neutered="0" and escape_attemts < '3' 
+/* List the date of birth of all animals named either "Agumon" or "Pikachu"*/
+SELECT DISTINCT date_of_birth FROM animals WHERE name="Agumon" or name="Pikachu"  
+/* List name and escape attempts of animals that weigh more than 10.5kg */
+SELECT DISTINCT name, date_of_birth FROM animals WHERE weight_kg > "10.5"  
+/* Find all animals that are neutered. */
+SELECT * FROM animals WHERE neutered = "0"
```

Comment:
* Kindly use the right data type for `neutered` which is Boolean [check project requirements here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md#:~:text=Create%20a%20table%20named,weight_kg%3A%20decimal)